docs/containers/debug-python.md

```diff
@@ -0,0 +1,149 @@
+---
+Area: containers
+ContentId: f9ffec31-9253-4f71-a4eb-79ea7b3a8f55
+PageTitle: Configure and troubleshoot debugging of Python apps running in a Docker container
+DateApproved: 01/29/2020
+MetaDescription: How to configure and troubleshoot debugging of Python apps running in a Docker container, using Visual Studio Code.
+---
+
+# Debug Python within a container
+
+When adding Docker files to a Python project, tasks and launch configurations are added to enable debugging the application within a Docker container. To accommodate the various scenarios of Python projects, some apps may require additional configuration.
+
+## Configuring the Docker container entry point
+
+The Docker extension infers the entry point of the Docker container via properties of `tasks.json`. This container entry point is automatically configured when using the **Docker: Add Docker Files to Workspace...** command.
```

Comment:
Trying to rephrase to make it more clear what applies to the user - is this right?
```suggestion
You can configure the entry point of the Docker container by setting properties in `tasks.json`. VS Code automatically configures the container entry point when you first use the **Docker: Add Docker Files to Workspace...** command.
```